<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>524</width>
      <height>396</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHide_finished_Tasks"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionNew_Task"/>
    <addaction name="actionPending_Task"/>
    <addaction name="actionFinished_Task"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Todo_Application"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Task"/>
   <addaction name="actionPending_Task"/>
   <addaction name="actionFinished_Task"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_Todo_Application"/>
   <addaction name="separator"/>
   <addaction name="actionHide_finished_Tasks"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/exit (3).png</normaloff>:/exit (3).png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Task">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/add_list.png</normaloff>:/add_list.png</iconset>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
  </action>
  <action name="actionPending_Task">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/pending.png</normaloff>:/pending.png</iconset>
   </property>
   <property name="text">
    <string>Pending Task </string>
   </property>
  </action>
  <action name="actionFinished_Task">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/taskdone.png</normaloff>:/taskdone.png</iconset>
   </property>
   <property name="text">
    <string>Finished Task </string>
   </property>
  </action>
  <action name="actionAbout_Todo_Application">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/Aboutt.png</normaloff>:/Aboutt.png</iconset>
   </property>
   <property name="text">
    <string>About Todo Application</string>
   </property>
  </action>
  <action name="actionHide_finished_Tasks">
   <property name="icon">
    <iconset resource="TodoIcons.qrc">
     <normaloff>:/hide (2).png</normaloff>:/hide (2).png</iconset>
   </property>
   <property name="text">
    <string>Hide finished Tasks </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="TodoIcons.qrc"/>
 </resources>
 <connections/>
</ui>
